.spelling

```diff
@@ -123,4 +123,12 @@ WinGet
 WinRT
 WinUI
 WinUI3
-Xcode
\ No newline at end of file
+Xcode
+sidebar_label

```

Comment:
This shouldn't be necessary

Reply:
I pushed a commit to fix this - those tags should be ignored by the spell checker, except on those new pages we have `:` characters in the title name, so they're wrapped in quotes, which the version bump lost. Probably a bug they fixed in newer docusauruses.